src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -303,6 +311,35 @@ def perform_destroy(self, instance):
     destroy=extend_schema(
         summary="Verwijder een onderwerpobject.",
         description="Verwijder een onderwerpobject.",
+        parameters=[
+            OpenApiParameter(
+                name="cascade",
+                description=(
+                    "Als `true`, worden gerelateerde Klantcontacten en DigitaalAdressen verwijderd "
+                    "indien ze niet door andere Onderwerpobjecten worden gebruikt."
+                ),
+                required=False,
+                default=False,
+                type=bool,
+                location=OpenApiParameter.QUERY,
+            ),
+        ],
+        responses={
+            status.HTTP_204_NO_CONTENT: OpenApiResponse(
+                description="Onderwerpobject succesvol verwijderd."
+            ),
+            status.HTTP_200_OK: OpenApiResponse(
+                description=(
+                    "Onderwerpobject verwijderd met cascade=true, maar één of meer "
+                    "Klantcontacten worden nog door andere Onderwerpobjecten gebruikt."
+                ),
+                response=ListField(
+                    child=URLField(
+                        help_text="URL van een overgebleven Klantcontact resource"
+                    )
+                ),
```

Comment:
This doesn't seem to be picked up correctly by drf-spectacular and redoc currently:

<img width="827" height="399" alt="image" src="https://github.com/user-attachments/assets/90b6e942-f310-43bb-a7b6-b34352b481dc" />

I think you might have to use a serializer from `rest_framework` (or define one for this)